src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccessController.php

```diff
@@ -82,11 +84,15 @@ public function createAction(Request $request): Response
 
         try {
             $handlerResult = $this->getFormHandler()->handle($apiAccessForm);
-
             if (null !== $handlerResult->getIdentifiableObjectId()) {
-                $this->addFlash('success', $this->trans('Successful creation', 'Admin.Notifications.Success'));
+                /** @var CreatedApiAccess $apiAccess */
+                $apiAccess = $handlerResult->getIdentifiableObjectId();
+                $this->displayTemporarySecret(
+                    $this->trans('The API access and Client secret have been generated successfully.', 'Admin.Notifications.Success'),
```

Comment:
@jolelievre Is it a toast notification?
If yes, maybe something shorter would be better so the user has time to read. For example 

'API access and client secret generated'

Reply:
No, it's a notification message that only closes when the user decides to:

![Capture d’écran 2023-10-27 à 10 31 15](https://github.com/PrestaShop/PrestaShop/assets/13801017/c49dfbe2-0e8c-4199-9182-ca81d25ce9de)

Reply:
Ok then, I made a suggestion. Thank you for the screenshot.